.github/config.yml

```diff
@@ -0,0 +1,25 @@
+
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  Thanks for opening your first issue here! Please take some time to explain the issue and help us help you 👍🎉😄
+  If you have screenshots to share demonstrating the issue, that's really helpful! 📸
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+ Thanks for opening this pull request! The tests defined in `builder/test` will test out your code on every commit.
+ There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: |
+ Congrats on merging your first pull request! 🙌🎉⚡️ 
+ Your code will likely be published to PublicLab.org in the next few days. 
```

Comment:
Yes! However note here where the help text is not relevant to this project. Could you imagine a different text that would make more sense for this project instead of PublicLab.org? Thanks!

Reply:
@jywarren, 
I think the code can be related to documentation, enhancement or bug. So, to generalize it I will suggest replacing the line `Your code will likely be published to PublicLab.org in the next few days.` to `Your code will help in making this project better.` Will that is okay 😄